src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,153 @@
 package core.basesyntax;
 
+//import java.util.HashMap;
+import java.util.Objects;
+import java.util.Random;
+
```

Comment:
Only one line should separate import statements and top-level class declaration. So please remove one of these blank lines. 
Ref: https://mate-academy.github.io/style-guides/java/java.html#s3-source-file-structure